### PR TITLE
add model quality scores

### PR DESCRIPTION
**Pull request by bokelley**

### CODE REVIEW

File: docs/snippets/defaults_channel_mapping.mdx [hunk 1]
```diff
@@ -35,4 +35,32 @@ default_platform_ad_format_by_channel:
...
+  app:      1
+  web:      1
+default_property_ad_funded_percentage_by_channel:
+  dooh:     1
+  social:   1
+  ctv-bvod: 1
+  audio:    1
+  app:      1
+  web:      1
+default_average_seconds_per_session_excluding_ads_by_channel:
```
Comment (EmmaLouise2018) [inline docs/snippets/defaults_channel_mapping.mdx:52]: - [ ] populate defaults 

Comments:
- We have decent session time data from similarweb/data.ai so we can pull an 80th percentile number and contribute. Feels like 100% is the right ad funded default (to be conservative), and we can pull same imps per session numbers that we had before (I think this is really a "imps per session second" default like we have now?)

File: docs/snippets/defaults_channel_mapping.mdx [hunk 2]
```diff
@@ -35,4 +35,32 @@ default_platform_ad_format_by_channel:
...
+  dooh:     1
+  social:   1
+  ctv-bvod: 1
+  audio:    1
+  app:      1
+  web:      1
+default_average_data_kb_per_session_seconds_excluding_ads_by_channel:
```
Comment (EmmaLouise2018) [inline docs/snippets/defaults_channel_mapping.mdx:59]: - [ ] populate defaults 

Comments
- I think for CTV/audio this comes from the average bitrate, probably. For app-based channels let's pull an 80th percentile number based on what we have from data.ai... I think we could probably get web from a sampling of archive.org if we want, or can take 80th percentile of our database? This one would be helped with Sincera data prob.

File: docs/model_quality.mdx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+- Organization model quality (1-5) 
+- Property model quality (1-5)
+- Ad stack / ad platform quality (1-5)
+- Input granularity score (1-5)
+
+## Grid mix
+
+Grid intensity can fluctuate significantly on an hourly basis due to the variable nature of both renewable energy sources (sun, wind) and electricity demand. To achieve effective decarbonization, having hourly data is critical. However, this isn't broadly available in many countries.
+
+| Timescale | Geographical fit | Model Quality |
```
Comment (MikeFreyberger) [inline docs/model_quality.mdx:16]: how do you think we should represent the model quality for countries where we do not get any country specific grid mix information and we need to fallback to a WW model? i'd think that should be lower than annual + country

File: docs/model_quality.mdx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+| 30% - 50%     | 2             |
+| 50% - 70%     | 3             |
+| 70% - 90%     | 4             |
+| 90% - 100%    | 5             |
+
+Example:
+
+| Field           | Provided Input | Match   |
+| --------------- | -------------- | ------- |
+| Channel         | web            | yes     |
```
Comment (MikeFreyberger) [inline docs/model_quality.mdx:94]: generally a given inventory identifier resolves to a single channel (and given the difficulty of mapping traditional reports to the channel column) I'd be hesitant to penalize not providing channel.

also, other input cols that I'd think we would want to consider when measuring input granularity score:
- country
- region (if relevant based on country)
- device type
- network

File: docs/model_quality.mdx (hunk 3 above)
Comment (bokelley) [reply docs/model_quality.mdx:16]: updated to reflect this feedback

File: docs/model_quality.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
+To effectively understand how accurate an emissions "measurement" is, given the many layers of assumptions and allocations involved in the calculation, is critical both for assessing provider quality and as a gating factor to incorporate data into corporate and regulatory reporting.
```
Comment (EmmaLouise2018) [inline docs/model_quality.mdx:1]: nit: `in the calculation, **it** is critical both ..`

File: docs/model_quality.mdx [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+| Report element                  | Model Quality |
+| ------------------------------- | ------------- |
+| Scope 1 and 2, market-based     | +1 |
+| Location-based scope 2          | +1 |
+| All scope 3 categories provided | +1 |
+| Lines of business broken out    | +1 |
+
+## Vendor / Ad Platform Quality
+
+An ad platform should provide 1) requests received, 2) requests sent (traffic shaping), and 3) emissions data from datacenters or cloud providers
```
Comment (EmmaLouise2018) [inline docs/model_quality.mdx:44]: You out this in another section, and I think its good to be explicit. Add `The vendor / ad platform score starts at 1 and increments based on modeled components.`

**Review by EmmaLouise2018 — APPROVED** [review]

**Review by MikeFreyberger — APPROVED** [review]

**Pull request merged**